README.md

```diff
@@ -23,6 +23,13 @@ Third, download (or update) and build Sigourney and its dependencies:
 
 	$ go get -u github.com/nf/sigourney
 
+Fourth, install [GopherJS](https://github.com/gopherjs/gopherjs):
+
+	$ go get -u github.com/gopherjs/gopherjs
+
+Fifth, build the frontend:
+
+	$ gopherjs build $GOPATH/src/github.com/nf/sigourney/static/script.go -o $GOPATH/src/github.com/nf/sigourney/static/script.js
```

Comment:
You could make this step easier by turning it into a `go:generate` directive, then you just need to run a standard `go generate ./...` command to update the package.